ORGANIZATION.md

```diff
@@ -0,0 +1,93 @@
+# OpenSearch Build / Release Ownership
+
+On OpenSearch there are a number of processes that could be handled by individual development teams - and we are fortunate that there are a number of dedicate team members that can centralize the effort to help these teams scale so smaller teams can keep focused on their areas.  This team is called the OpenSearch DevOps team, it is not exclusive, but it’s dedicated.
+
+This document’s purpose is to clarify where the boundaries between these projects exist and how to get things done.
+
+## Build
+
+OpenSearch is build on code and that code needs to build.  No specific build system is required; however, gradle and yarn are recommendations and will have the most support.
```

Comment:
The first sentence seems confusing. Shall we say "OpenSearch is built on code and that code needs to be built" to be more precise?

Reply:
"We need to build OpenSearch Distributions with the code committed to this repo?"

@zelinh's sentence still sounds confusing to me.